package.json

```diff
@@ -164,6 +165,7 @@
     "eslint": "^7.20.0",
     "eslint-plugin-jsx-control-statements": "^2.2.1",
     "eslint-plugin-react": "^7.19.0",
+    "fetch-mock": "^9.11.0",
```

Comment:
This is needed to mock requests made from `@opendatasoft/api-client` - which uses `fetch` API.
It is only a dev dependency